live_data/main.py

```diff
@@ -0,0 +1,30 @@
+import logging
```

Comment:
General comment, live_data is a bit vague for the package name. A much better name IMO is `live_data_subscriber` or `live_data_consumer` to denote the fact that this service is subscribing to external data sources.